examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    sim_task = sim.run(ahs_program, shots=len(batch_mapping)*shots).result()
</pre><p>This is not a task. <code>.result()</code> at the end makes sure that we get the results. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>